bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js

```diff
@@ -82,12 +83,9 @@ function valueParser(val) {
   }
 }
 
-export default function addUserSettings(settings) {
+export default function addUserSettings(meetingId, userId, settings) {
   try {
     check(settings, [Object]);
-
-    const { meetingId, requesterUserId: userId } = extractCredentials(this.userId);
```

Comment:
Provided feedback to @gustavotrott 
We'll redo the parameter pass so we don't need to disturb this part.